scaler/handlers.go

```diff
@@ -38,18 +38,39 @@ func (e *impl) IsActive(
 	ctx context.Context,
 	scaledObject *externalscaler.ScaledObjectRef,
 ) (*externalscaler.IsActiveResponse, error) {
+	lggr := e.lggr.WithName("IsActive")
+	host, ok := scaledObject.ScalerMetadata["host"]
+	if !ok {
+		err := fmt.Errorf("no 'host' field found in ScaledObject metadata")
+		lggr.Error(err, "ScaledObjectRef", scaledObject)
+		return nil, err
```

Comment:
the second parameter to `lggr.Error` is a message for context, and the ones after that should be in pairs of two, where the first of each pair is the name of the value, and the second is the value. So, the `lggr.Error` call will need to change. Here's something that makes a bit more sense to me. Of course, feel free to change as you see fit.


```suggestion
              err := fmt.Errorf("no 'host' field found in ScaledObject metadata")
		lggr.Error(err, "returning immediately from IsActive RPC call", "ScaledObject", *scaledObject)
		return nil, err
```

Also note that we're dereferencing the `scaledObject` var (prefixing with `*`) because it's passed as a pointer. IIRC, this isn't strictly necessary for the particular implementation of `logr.Logger` that we're using, but it's a best practice that might come in handy if/when we decide to change implementations.